modules/nested-tabs-html/widgets/nested-tabs-html.php

```diff
@@ -1185,7 +1186,7 @@ protected function render() {
 		}
 		?>
 		<div <?php $this->print_render_attribute_string( 'elementor-tabs' ); ?>>
-			<div class="e-n-tabs-heading">
+			<div class="e-n-tabs-heading" role="tablist">
```

Comment:
The @axe-core/playwright testing tool required that we wrap the 'role="tablist"' directly around the 'role="tab"' elements.